repos/fdbt-site/src/interfaces/index.ts

```diff
@@ -348,6 +348,14 @@ export interface BasicService {
     serviceDescription: string;
 }
 
+export interface ReturnService extends BasicService {
+    id: number;
```

Comment:
What do we need this ID for? It doesn't look like it's used inside the Netex.

Reply:
removed the ReturnService interface